### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Connect to Local Server_Screen Reader: Narrator/NVDA does not announce the page heading or description while "Connect to Local Server" page opens.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#102]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated:** MAS2.4.6

**Repro Steps:**
1. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2. Download and install "Microsoft IIS Administration" application.
3. Tab till "Skip This" button and Press Enter.
4. Verify that screen reader announces the heading or description of the page when it opens.

**Actual Result:**
Narrator/NVDA does not announce the page heading or description while "Connect to Local Server" page opens.

**Expected Result:**
When page opens, screen reader should announce like: "Connect to Local Server" along with any other required description.

**User Impact:**
Screen Reader users will be unable to know the information about the current page they are trying to access.

**Note:** This issue occurs throughout the application.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}

**Attachment:**
[Bug 102_screen reader is not announcing the heading or page description when the page opens.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323637/Bug.102_screen.reader.is.not.announcing.the.heading.or.page.description.when.the.page.opens.pptx)

**Comment by jhkimnew** [comment]
This issue is only for Site mode, which is now low priority.

**Comment by jhkimnew** [comment]
Closing because this UI is not used anymore.

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
